### ISSUE TITLE
Can't retrieve images from the Culturally Authentic Pictorial Lexicon (CAPL) provider

### ISSUE THREAD

**Issue by krysal** [WordPress/openverse#4791]
## Description

When trying to explore items from this provider/source, the API says the param value is invalid despite being included (last) in the exact list of valid options, making this provider of +15,000 images virtually unavailable.

```json
{
    "detail": {
        "source": [
            "Invalid source parameter 'CAPL'. No valid sources selected. Refer to the source list for valid options: 'wordpress', 'woc_tech', 'wikimedia', 'wellcome_collection', 'waltersartmuseum', 'thingiverse', 'svgsilh', 'stocksnap', 'spacex', 'smk', 'smithsonian_zoo_and_conservation', 'smithsonian_postal_museum', 'smithsonian_portrait_gallery', 'smithsonian_national_museum_of_natural_history', 'smithsonian_libraries', 'smithsonian_institution_archives', 'smithsonian_hirshhorn_museum', 'smithsonian_gardens', 'smithsonian_freer_gallery_of_art', 'smithsonian_cooper_hewitt_museum', 'smithsonian_anacostia_museum', 'smithsonian_american_indian_museum', 'smithsonian_american_history_museum', 'smithsonian_american_art_museum', 'smithsonian_air_and_space_museum', 'smithsonian_african_art_museum', 'smithsonian_african_american_history_museum', 'sketchfab', 'sciencemuseum', 'rijksmuseum', 'rawpixel', 'phylopic', 'nypl', 'national_museum_of_finland', 'nasa', 'nappy', 'museumsvictoria', 'met', 'justtakeitfree', 'inaturalist', 'geographorguk', 'floraon', 'flickr', 'finnish_satakunnan_museum', 'finnish_heritage_agency', 'europeana', 'east_riding', 'digitaltmuseum', 'clevelandmuseum', 'brooklynmuseum', 'bio_diversity', 'bib_gulbenkian', 'archief_alkmaar', 'animaldiversity', 'WoRMS', 'CAPL'."
        ]
    }
}
```
 
## Reproduction

1. Go to https://api.openverse.org/v1/images/?source=CAPL
2. See error.

**Comment by obulat** [comment]
"Hide content" in the admin is set to true for CAPL: _The domain was found to be unresponsive on 2024-08-20._ 

Should we update the response to "The source has been temporarily disabled" when the source is disabled in the admin?

**Comment by sarayourfriend** [comment]
I suppose we also need to fix the error message, CAPL shouldn't be in the list of available sources if it is disabled.

Are there two aspects to this issue? Fixing the error message shouldn't be blocked on the particular provider's issues, right? I can open a separate issue for fixing the error message, if that's the case.
